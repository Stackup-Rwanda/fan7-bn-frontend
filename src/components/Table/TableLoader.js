import React from 'react';
import './styles/TableLoader.scss';

const TableLoader = () => (
  <div className="table-ellipsis">
    <div></div>
    <div></div>
    <div></div>
    <div></div>
  </div>
);

export default TableLoader;
