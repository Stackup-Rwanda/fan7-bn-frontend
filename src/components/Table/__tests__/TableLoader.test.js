import React from 'react';
import { mount, shallow } from 'enzyme';
import TableLoader from '../TableLoader';

describe('TableLoader Component Test', () => {
  it('should return true if component exists', () => {
    const wrapper = mount(<TableLoader />);

    expect(wrapper.find('.table-ellipsis').exists()).toBe(true);
  });
});

describe('<TableLoader /> shallow rendering tests', () => {
  it('matches the snapshot', () => {
    const tree = shallow(<TableLoader />);
    expect(tree).toMatchSnapshot();
  });
});
