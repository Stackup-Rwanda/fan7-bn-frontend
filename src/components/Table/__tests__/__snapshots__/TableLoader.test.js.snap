// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TableLoader /> shallow rendering tests matches the snapshot 1`] = `
<div
  className="table-ellipsis"
>
  <div />
  <div />
  <div />
  <div />
</div>
`;
